librz/core/cmd_debug.c

```diff
@@ -4659,6 +4651,20 @@ static void consumeBuffer(RzBuffer *buf, const char *cmd, const char *errmsg) {
 	rz_cons_printf("\n");
 }
 
+RZ_IPI int rz_debug_continue_oldhandler(void *data, const char *input) {
+	RzCore *core = (RzCore *)data;
+	rz_reg_arena_swap(core->dbg->reg, true);
+#if __linux__
+		core->dbg->continue_all_threads = true;
+#endif
+	if (rz_debug_is_dead(core->dbg)) {
+		eprintf("Cannot continue, run ood?\n");
+		return -1;
```

Comment:
I suspect this has to do with this. Try to return 0 here. Unfortunately existing commands often just return a "success" return value even when something goes wrong. This is an interesting point to re-evaluate when we convert `d` subcommands to newshell.

Reply:
And 1 when it succeeds?

Reply:
No, 0/1 is the same IIRC. As said, this return mess 0 vs negative vs positive vs bool vs whatever would finally be resolved when switching to newshell, where we have well-defined return states https://github.com/rizinorg/rizin/blob/dev/librz/include/rz_cmd.h#L23